### ISSUE TITLE
Slockit Pi state disapears and errors

### ISSUE THREAD

**Issue by CodeDecent** [slockit/usn-mvp#10]
Slockit Pi
LG G6
Android 8

Finally connects to Pi but now when youre about to set it up it crashes and clicking back button closes the app. Reopen the app and ui/state disapears. Clicking link smart object below Pi option it gives an alert the function doesnt exist.

**Comment by nomoreboredom** [comment]
@CodeDecent 

Thanks again for your input :)
Regarding the 'undefined' alert, you can ignore it, as 'Link Smart Object' CTA should not be there at the first place. Linking smart object for the Pi can be done after tapping the peer on the same Access tab, and there you will find a button to 'Link Smart Object'. 

Regarding, 'when youre about to set it up it crashes and clicking back button closes the app. Reopen the app and ui/state disapears.' Can you clarify this, when exactly does this happen?  

Normally the flow is the following. After completing the peer setup, it writes to local storage the peer Pi host, name, and such data. Then entering Access screen initiates a request to read Pi. Can you let me know in that tab if you see 'Pi is not accessible' or 'No device set'. 

Thanks!

**Comment by CodeDecent** [comment]
Yes latest version. Pi not accessible. Clicked on that then a blank screen after.

**Comment by CodeDecent** [comment]
After crash i reoppened app and got this.

**Comment by Quaternioneer** [comment]
On CodeDecent's first post notice that the subnet ip 10.42.0.1 precludes the possibility of 255.255.255.0 subnet mask.   So, I presume that someone hardcoded the default gateway and a subnet mask of 255.0.0.0 on which Slockit Pi was originally tested.  If you could fix this, it would really in diagnosing if there are networking issues.

I can confirm this bug on 
Samsung S8
Slockit Pi 2nd version February 2018 

The very last picture above which is blank has a "Link Smart Object"  button at the bottom before going blank.
